package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sw-app",
+  "version": "0.0.1",
+  "repository": "git@github.com:rafalgolubowicz/sw-app.git",
+  "author": "Rafal Golubowicz",
+  "license": "MIT",
+  "private": true,
+  "workspaces": [
+    "packages/backend",
+    "packages/frontend"
+  ],
+  "scripts": {
+    "start:api": "yarn --cwd packages/backend dev",
+    "start:ui": "yarn --cwd packages/frontend start"
+  },
+  "dependencies": {
+    "@types/node": "^17.0.19",
+    "@typescript-eslint/eslint-plugin": "^5.12.1",
+    "@typescript-eslint/parser": "^5.12.1",
+    "eslint": "^8.9.0",
+    "eslint-config-prettier": "^8.4.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "prettier": "^2.5.1",
+    "typescript": "^4.5.5"
```

Comment:
should not most of these, if not all be `devDependencies`?

Reply:
indeed!